packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -52,9 +52,12 @@ export class Prerendering {
             this.wafermap.colorScale,
             this.wafermap.colorScaleMode
         );
+        const isDieRenderInfo = (
+            info: DieRenderInfo | null
+        ): info is DieRenderInfo => info !== null;
         this._diesRenderInfo = this.wafermap.dies
-            .map(die => this.computeDieRenderInfo(die))
-            .filter(info => info !== null) as DieRenderInfo[];
+            ?.map(die => this.computeDieRenderInfo(die))
+            .filter(isDieRenderInfo) ?? [];
```

Comment:
Hopefully after dies is restored to not being nullable this can be simplified to

```suggestion
            .map(die => this.computeDieRenderInfo(die))
            .filter(isDieRenderInfo);
```